core/jobs/job_options.py

```diff
@@ -63,6 +63,8 @@ def __init__(
             region=feconf.GOOGLE_APP_ENGINE_REGION,
             temp_location=feconf.DATAFLOW_TEMP_LOCATION,
             staging_location=feconf.DATAFLOW_STAGING_LOCATION,
+            experiments=['use_runner_v2', 'enable_recommendations'],
```

Comment:
Would be good to have some comments here explaining what's going on.

Reply:
Done.

Reply:
Done.